server/routes/api-v1/organizationRouter.ts

```diff
@@ -1,8 +1,10 @@
 import express from 'express';
-import * as authController from '../../controllers/authController';
+import * as MSController from '../../controllers/MSController';
 
 const organizationRouter = express.Router();
 
-organizationRouter.get('/logo', authController.getAzureADApplicationLogo);
+organizationRouter.get('/logo', MSController.getAzureADApplicationLogo);
+organizationRouter.get('/users', MSController.getAllUsers);
+organizationRouter.get('/users/:id/photo', MSController.getUserPhoto);
```

Comment:
We need some standard. I used to use singular words for naming resources, but i would like to change this in other resources to plural words, so i can add this as a task to backlog or change this in this branch